index.html

```diff
@@ -391,7 +393,8 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
 
                         <div class="form-group checkbox">
                             <label for="officialDriver">
-                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" /> I am driving in an official capacity on behalf of a registered organization
+                              <input type="checkbox" class="hidden" name="V" value="B" style="display:none;">
+                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" style="display:none;" class="hidden" /> <!--I am driving in an official capacity on behalf of a registered organization -->
```

Comment:
I'm not sure that this is what was intended, but I'll need to confirm with @sasjkia what she meant when she said to hide this checkbox. I assume she meant to delete this form row. Sorry, I'll get back to you :(